apps/nocobase/1.4.4/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   nocobase:
-    image: nocobase/nocobase:1.4.4
+    image: nocobase/nocobase:1.4.5
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
I have no information that would allow me to analyze this specific code snippet provided here. Please give more context about which specific code you want a review on!